app/helpers/format-markdown.js

```diff
@@ -1,5 +1,6 @@
-import Ember from "ember";
-var showdown = new Showdown.converter();
-export default Ember.Handlebars.makeBoundHelper(function(value) {
-  return new Ember.Handlebars.SafeString(showdown.makeHtml(value));
-});
\ No newline at end of file
+import Ember from 'ember';
+
+export default Ember.Helper.helper(function(params) {
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).